activesupport/test/core_ext/duration_test.rb

```diff
@@ -760,6 +760,16 @@ def test_duration_symmetry
     assert_equal expected_time, time + ActiveSupport::Duration.build(-1)
   end
 
+  def test_duration_negative
+    assert_not 1.hours.negative?
+    assert_not 1.days.negative?
+    assert_not (1.weeks - 1.days).negative?
```

Comment:
There is no rubocop rule for that but generally for the same reason why we prefer `assert_predicate` over `assert` we should be using `assert_not_predicate` as it produces a better error message. Even though `assert_not_predicate` may not sound very natural I think having better test failure message overweights 

```suggestion
    assert_not_predicate 1.hours, :negative?
    assert_not_predicate 1.days, :negative?
    assert_not_predicate (1.weeks - 1.days), :negative?
```